instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -78,6 +120,30 @@ public final class InstrumenterBuilder<REQUEST, RESPONSE> {
     this.spanNameExtractor = spanNameExtractor;
     this.instrumentationVersion =
         EmbeddedInstrumentationProperties.findVersion(instrumentationName);
+
+    List<ContextCustomizerCustomizer> contextProviders =
+        CONTEXT_CUSTOMIZER_MAP.get(instrumentationName);
+    if (contextProviders != null) {
+      for (ContextCustomizerCustomizer provider : contextProviders) {
+        addContextCustomizer(provider.get());
+      }
+    }
+
+    List<AttributesExtractorCustomizer> attributeProviders =
```

Comment:
[for review] If an AttributesExtractor with the same key as the vanila Java Agent appears, and the custom one currently has a lower priority, do we need to make some adjustments?

Reply:
could move this code to the `build()` method so that they are added at the end of the list